gcp/app-engine/test_main.py

```diff
@@ -55,108 +55,108 @@ def test_json_serial_int():
-    mock_execute = mock_conn.execute
-    mock_execute.return_value.fetchall.return_value = mock_records
-
-    # Make the request to the API
-    response = client.get('/api/v1/tournaments')
-
-    # Check that execute was called with the correct query
-    expected_query = f"SELECT * FROM tournaments where 1=1 and id = {m.args['id']}"
-    actual_query = mock_execute.call_args[0][0].text
-    assert actual_query == expected_query, f"Expected query: {expected_query}, but got: {actual_query}"
-
-    # Check the response data
-    expected_response = [
-        {'id': 1, 'name': 'Tournament 1'},
-        {'id': 2, 'name': 'Tournament 2'}
-    ]
-    assert response.status_code == 200
-    assert json.loads(response.data) == expected_response
-
-def test_get_players_no_args(client, mocker):
-    # Mock get_db_connection
-    mock_db_connection = mocker.patch('main.get_db_connection')
-    mock_pool = mock_db_connection.return_value
-    mock_conn = mock_pool.connect.return_value.__enter__.return_value
-    mock_execute = mock_conn.execute
-    mock_execute.return_value.fetchall.return_value = mock_records
-
-    # Make the request to the API
-    response = client.get('/api/v1/players')
-
-    # Check that execute was called with the correct query
-    expected_query = "SELECT * FROM tournament_players where 1=1"
-    actual_query = mock_execute.call_args[0][0].text
-    assert actual_query == expected_query, f"Expected query: {expected_query}, but got: {actual_query}"
-
-    # Check the response data
-    expected_response = [
-        {'id': 1, 'name': 'Tournament 1'},
-        {'id': 2, 'name': 'Tournament 2'}
-    ]
-    assert response.status_code == 200
-    assert json.loads(response.data) == expected_response
-
-def test_get_players_with_id(client, mocker):
-    # Mock request
-    m = mocker.MagicMock()
-    m.args = {'id': 77, 'tournamentId': 88}
-    mocker.patch("main.request", m)
-
-    # Mock get_db_connection
-    mock_db_connection = mocker.patch('main.get_db_connection')
-    mock_pool = mock_db_connection.return_value
-    mock_conn = mock_pool.connect.return_value.__enter__.return_value
-    mock_execute = mock_conn.execute
-    mock_execute.return_value.fetchall.return_value = mock_records
-
-    # Make the request to the API
-    response = client.get('/api/v1/players')
-
-    # Check that execute was called with the correct query
-    expected_query = f"SELECT * FROM tournament_players where 1=1 and tournament_id = {m.args['tournamentId']} and id = {m.args['id']}"
-    actual_query = mock_execute.call_args[0][0].text
-    assert actual_query == expected_query, f"Expected query: {expected_query}, but got: {actual_query}"
-
-    # Check the response data
-    expected_response = [
-        {'id': 1, 'name': 'Tournament 1'},
-        {'id': 2, 'name': 'Tournament 2'}
-    ]
-    assert response.status_code == 200
-    assert json.loads(response.data) == expected_response
+# def test_get_tournaments_no_args(client, mocker):
```

Comment:
comment out tests for now until we migrate over to the new endpoints (now that i've removed the sportsdataio endpoints). will add back later